CHANGELOG.md

```diff
@@ -69,6 +69,35 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 - **(breaking)** Text rendering is now implemented using `Text`, `TextStyle` and `Font` objects instead of using `Font::render_str`.
 
+- **(breaking)** #224 Embedded Graphics macros now use named instead of positional parameters
+
+  ```rust
+  // OLD
+  egcircle!((CIRCLE_SIZE, CIRCLE_SIZE), CIRCLE_SIZE as u32);`
+  // NEW
+  egcircle!((CIRCLE_SIZE, CIRCLE_SIZE), CIRCLE_SIZE as u32);
```

Comment:
The parameter names are missing and the example should also use number literals instead of `CIRCLE_SIZE` IMO.